src/main/java/com/findmyflight/findmyflight/service/emailnotificationreceiver/EmailNotificationReceiver.java

```diff
@@ -0,0 +1,25 @@
+package com.findmyflight.findmyflight.service.emailnotificationreceiver;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.validation.constraints.NotBlank;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class EmailNotificationReceiver {
+    @Id
+    @Column(unique = true)
```

Comment:
`@Id` column is always unique, as it is automatically incremented and assigned by a database itself when inserting a row - we need `@Column(unique = true)` annotation on `address` field